pkg/extensions/tbtc/tbtc.go

```diff
@@ -604,6 +616,28 @@ func (t *tbtc) watchKeepClosed(
 	return signalChan, unsubscribe, nil
 }
 
+func (t *tbtc) shouldMonitorDeposit(
+	depositAddress string,
+) bool {
```

Comment:
I think this one as well as `Initialize` in this file and `CreateDeposit` in `pkg/chain/local/tbtc.go` should fit in one line. We don't need to break them.

Reply:
Agree with the first two but `CreateDeposit` doesn't fit.